src/julia_gc.c

```diff
@@ -387,6 +388,36 @@ void MarkJuliaObj(void * obj)
 // from which freed objects are then marked. Hence, we add additional checks
 // when traversing GAP master pointer and bag objects that this happens
 // only for live objects.
+//
+// All stacks in Julia are associated with tasks and we can use
+// jl_task_stack_buffer() to retrieve the buffer information for that stack.
+// That said, in a couple of cases these need adjustments.
+//
+// 1. The stack buffer of the root task of the main thread, when started from
+//    GAP can extend past the point where Julia believes it ends. Therefore,
```

Comment:
Shouldn't this be:
```suggestion
//    GAP can extend past the point where Julia believes it starts. Therefore,
```
or
```suggestion
//    GAP can extend past the point where Julia believes its origin is. Therefore,
```
or so?

Reply:
I like the second option best.